maat-court-data-api/src/main/java/gov/uk/courtdata/billing/controller/ApplicantResetController.java

```diff
@@ -0,0 +1,33 @@
+package gov.uk.courtdata.billing.controller;
+
+import gov.uk.courtdata.billing.request.ApplicantResetRequest;
+import gov.uk.courtdata.billing.service.ApplicantResetService;
+import gov.uk.courtdata.eform.controller.StandardApiResponseCodes;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@Tag(name = "Applicant CCLF Reset", description = "Rest API for resetting SEND_TO_CCLF flag in applicants")
+@RequestMapping("${api-endpoints.billing-domain}/applicant")
+public class ApplicantResetController {
+
+    private final ApplicantResetService applicantResetService;
```

Comment:
I was going to ask if you could rename all of these classes that start `ApplicantReset` to be something more like `ApplicantBilling ...`, but then I realised that we do _already_ have a controller in the code for dealing with applicant billing requests. You may have missed it as it's naming is not consistent with the others - it's called `BillingApplicantController`; this is what we should be using for the applicant logic, with the _reset_ being one endpoint on that controller.